.github/workflows/publish_packages.yml

```
@@ -0,0 +1,62 @@
+name: publish_packages.yml
+on:
+  release:
+    types:
+      - released
```

Following the discussion from different comment 

```suggestion
      - published
```